frontend/public/locales/ja/network-policy.json

```diff
@@ -9,7 +9,7 @@
   "NS selector": "NS セレクター",
   "NetworkPolicy details": "ネットワークポリシーの詳細",
   "Ingress rules": "Ingress ルール",
-  "Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy. See more details in:": "Pod はデフォルトですべてのトラフィックを受け入れます。これらは、ingress ルールのホワイトリストを指定するネットワークポリシー経由で分離できます。Pod がネットワークポリシーによって選択されると、ネットワークポリシーで明示的に許可されていないすべてのトラフィックが拒否されます。詳細については、以下を参照してください。",
+  "Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy. See more details in: ": "Pod はデフォルトですべてのトラフィックを受け入れます。これらは、ingress ルールのホワイトリストを指定するネットワークポリシー経由で分離できます。Pod がネットワークポリシーによって選択されると、ネットワークポリシーで明示的に許可されていないすべてのトラフィックが拒否されます。詳細については、以下を参照してください。",
```

Comment:
There shouldn't be changes in the ja/ko/zh files, only en. The other files will be updated when we get translations back.